include/vast/Translation/CodeGenFallBackVisitor.hpp

```diff
@@ -31,21 +32,54 @@ namespace vast::cg
         }
     };
 
-    template< typename Derived >
+    template< typename Derived
+        , template< typename > typename FallBackMixin = UnsupportedFallBackVisitor
```

Comment:
`Unsupported` should not be anywhere in this class -> it should be plugged into the "dispatcher" class below as a template argument only (no extra type aliases or anything should be added to it).